runtime/onert/core/src/ir/operation/BCQFullyConnected.cc

```diff
@@ -31,7 +31,7 @@ void BCQFullyConnected::accept(OperationVisitor &v) const { v.visit(*this); }
 
 BCQFullyConnected::BCQFullyConnected(const OperandIndexSequence &inputs,
                                      const OperandIndexSequence &outputs, const Param &param)
-    : Operation{OperandConstraint::createExact(4u), inputs, outputs}, _param{param}
+    : Operation{OperandConstraint::createExact(5u), inputs, outputs}, _param{param}
```

Comment:
I don't know well about runtime so I have a question..!
Does `createExact(5u)` allow to get only four inputs?

Reply:
@llFreetimell Yes, I will checkout